README.md

```diff
@@ -2,8 +2,6 @@
 Let's collect the data of Woman on GitHub, who inspired you to code more, learn more.
 
 **Note:** Please maintain the below list in alphabetical order.
-
-[alicetragedy](https://github.com/alicetragedy)
```

Comment:
@sara-02 Actually @tapasweni-pathak pointed out that "Why?" might have come off as a little rude. I sincerely apologize. I really didn't mean to sound rude. Hope you understand.